incubator/jenkins-operator/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 appVersion: 0.3.0
-description: A Helm chart for Kubernetes Jenkins Operator
+description: "DEPRECATED: A Helm chart for Kubernetes Jenkins Operator"
+deprecated: true
```

Comment:
From number 3 in [PROCESSES.md#deprecating-a-chart](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart):

> Above the deprecated property add a comment noting that the chart is deprecated and linking to the deprecation policy.